tests/test_schema_rules.py

```diff
@@ -26,3 +26,11 @@ def test_lowercase_ids():
             assert (
                 schema["id"] == schema["id"].lower()
             ), "The id in {} should be lower case".format(schema["id"])
+
+
+def test_id_length():
+    for schema in dictionary.schema.values():
+        if "id" in schema:
+            assert (
+                len(schema["id"]) <= 63
```

Comment:
the limit is not exactly 63 - for example in the ticket, the node id is `peritraumatic_dissociation_experiences_questionnaire` but the error comes from `node_peritraumaticdissociationexperiencesquestionnaire_node_id_idx` being longer than 63 chars.

please check the ticket comments, they contain useful details